include/fastdds/dds/xtypes/type_representation/ITypeObjectRegistry.hpp

```diff
@@ -18,7 +18,7 @@
  */
 
 #include <fastdds/dds/xtypes/type_representation/TypeObject.hpp>
-
+#include <fastdds/dds/core/ReturnCode.hpp>
```

Comment:
Please, sort alphabetically.

I am also missing the `<string>` header

Reply:
Fixed in [b660c7e](https://github.com/eProsima/Fast-DDS/pull/4140/commits/b660c7e19c132624708e6e06a7b989926917aff7)